test_pull_requests

```diff
@@ -1183,6 +1354,7 @@ popd
         # we need to take the correct external actions if necessary and update the bot
         # comment to reflect the new state
         if submit_test_job
+          delete_comment_with_prefix(id, base_repo, flake_denied_prefix(base_repo, settings['name']), comments)
```

Comment:
It could be sitting in the merge queue for hours waiting to get to this point though.